deployment/postgres-init-db/sql/expansion/tables/expansion_set.sql

```diff
@@ -0,0 +1,20 @@
+create table expansion_set (
+  id integer generated always as identity,
+  command_dict_id integer not null,
+  mission_model_id integer not null,
+
```

Comment:
I think we probably want the expansion set to have references to each of the expansions in the set.

Reply:
Sounds like we might want a join table to establish this relationship. Something like `expansion_set_membership` with two columns `expansion_set_id` and `expansion_rule_id`.

Reply:
`expansion_set.sql` has this reference. Let me know if that is what you were looking for? This schema is the expansion configuration used when you expand activities in the plan. The `package/bundle` idea we have talked about in our meetings

Reply:
I schedule a slack huddle to discuss this further with folks.

Reply:
Be sure to update this thread with the result of that huddle!

Reply:
After the meeting, we went with the many-to-many approach between `expansion_set` and `expansion_rule` and will be using a join table (`expansion_set_to_rule`) to link these two tables together. This will allow for users to update the mission model or command dictionary and select which expansion logic to carry over to the new set.